minikube-gradle-plugin/src/test/resources/mockito-extensions/org.mockito.plugins.MockMaker

```diff
@@ -0,0 +1 @@
+mock-maker-inline
```

Comment:
Nit : looks like there's no new line at the end of this file?

Reply:
Okay added the newline.